src/screens/DashboardScreen.tsx

```diff
@@ -78,87 +96,101 @@ export default function DashboardScreen(): JSX.Element {
-                    Change Username
-                </InputLabel>
-                <FilledInput
-                    name='username'
-                    onChange={(e) => {
-                        setUsername(e.target.value);
-                    }}
-                    inputProps={{ minLength: 3 }}
+        <section>
+            <div aria-live='polite' className='flex flex-col items-center justify-center '>
+                <div>
+                    <Typography>Email: {session?.user.email}</Typography>
+                    <Typography>Username: {profile?.username}</Typography>
+                </div>
+                <FormControl sx={{ m: 0.5 }} variant='filled'>
+                    <InputLabel htmlFor='username' color='secondary' className='!text-text'>
+                        Change Username
+                    </InputLabel>
+                    <FilledInput
+                        name='username'
+                        onChange={(e) => {
+                            setUsername(e.target.value);
+                        }}
+                        inputProps={{ minLength: 3 }}
+                        sx={{ m: 0.5 }}
+                    />
+                </FormControl>
+                <Button
+                    variant='contained'
+                    type='button'
+                    color='secondary'
+                    startIcon={<Edit />}
                     sx={{ m: 0.5 }}
-                />
-            </FormControl>
-            <Button
-                variant='contained'
-                type='button'
-                color='secondary'
-                startIcon={<Edit />}
-                sx={{ m: 0.5 }}
-                onClick={() => changeUsername()}
-            >
-                Update Profile
-            </Button>
-            <FormControl sx={{ m: 0.5 }} variant='filled'>
-                <InputLabel htmlFor='country' color='secondary' className='!text-text'>
+                    onClick={() => changeUsername()}
+                >
+                    Update Profile
+                </Button>
+                <FormControl sx={{ m: 0.5 }} variant='filled'>
+                    <InputLabel htmlFor='country' color='secondary' className='!text-text'>
+                        Change Country
+                    </InputLabel>
+                    <FilledInput
+                        name='country'
+                        onChange={(e) => {
+                            setCountry(e.target.value);
+                        }}
+                        inputProps={{ maxLength: 2, minLength: 2 }}
+                        sx={{ m: 0.5 }}
+                    />
+                </FormControl>
+                <Button
+                    variant='contained'
+                    type='button'
+                    color='secondary'
+                    sx={{ m: 0.5 }}
+                    onClick={() => changeCountry()}
+                    startIcon={<Language />}
+                >
```

Comment:
Out of scope of this PR but I think we might want to make some custom button wrappers. All of these buttons could just be `<ContainedButton />`

Reply:
Created #456